tests/e2e/runner/__tests__/minter-suite-v2/ram-lib.test.ts

```diff
@@ -510,41 +543,91 @@ describe("RAMLib event handling", () => {
       expect(bidRes.updatedAt).toBe(auctionBid3Timestamp.toString());
 
       // Artist reduces auction length time
-      const tx4 = await minterRAMV0Contract.connect(artist).reduceAuctionLength(
-        currentProjectNumber, // _projectId
-        genArt721CoreAddress, // _coreContract
-        reducedTargetAuctionEnd // _auctionTimestampEnd
-      );
-      await tx4.wait();
+      const reduceAuctionLengthTx = await minterRAMV0Contract
+        .connect(artist)
+        .reduceAuctionLength(
+          currentProjectNumber, // _projectId
+          genArt721CoreAddress, // _coreContract
+          reducedTargetAuctionEnd // _auctionTimestampEnd
+        );
+      await reduceAuctionLengthTx.wait();
+
       await waitUntilSubgraphIsSynced(client);
-      const targetId = `${minterRAMV0Address.toLowerCase()}-${genArt721CoreAddress.toLowerCase()}-${currentProjectNumber}`;
-      const minterConfigRes = await getProjectMinterConfigurationDetails(
+
+      // validate extraMinterDetails
+      minterConfigRes = await getProjectMinterConfigurationDetails(
         client,
         targetId
       );
-      // validate extraMinterDetails
-      const extraMinterDetails = JSON.parse(minterConfigRes.extraMinterDetails);
+
+      extraMinterDetails = JSON.parse(minterConfigRes.extraMinterDetails);
       expect(extraMinterDetails.auctionEndTime).toBe(reducedTargetAuctionEnd);
+      // The configuredAuctionEndTime is only updated when receiving an AuctionConfigUpdated event,
+      // not when receiving AuctionTimestampEndUpdated. While AuctionTimestampEndUpdated is emitted
+      // in three cases:
+      // 1. When a bid in the last 5 minutes extends the auction (most common)
+      // 2. When reduceAuctionLength is called
+      // 3. When adminAddEmergencyAuctionHours is called
+      // Ideally, we would update configuredAuctionEndTime for cases 2 and 3. However, since
+      // these are edge cases and we cannot reliably differentiate them from the more common
+      // case 1, we err on the side of not updating configuredAuctionEndTime in the
+      // AuctionTimestampEndUpdated handler.
```

Comment:
recommend a similar comment in `ram-lib-mapping::handleAuctionTimestampEndUpdated`

Reply:
Done!